lib/Interpreter/BackendPasses.cpp

```diff
@@ -69,6 +70,52 @@ namespace {
 
 char KeepLocalGVPass::ID = 0;
 
+namespace {
+
+  // Add a suffix to the CUDA module ctor/dtor to generate an unique name.
+  // It's necessary for lazy compilation. Without suffix, cling can't
```

Comment:
It's -> This is

Do not use appreviated in written language, e.g. can't -> "cannot"